Doc/library/bisect.rst

```diff
@@ -140,10 +142,10 @@ thoughts in mind:
 Searching Sorted Lists
 ----------------------
 
-The above :func:`bisect` functions are useful for finding insertion points but
-can be tricky or awkward to use for common searching tasks. The following five
-functions show how to transform them into the standard lookups for sorted
-lists::
+The above :py:mod:`bisect functions <bisect>` are useful for finding insertion
```

Comment:
What about doing something like this?

```diff
diff --git a/Doc/library/bisect.rst b/Doc/library/bisect.rst
index e3c8c80190..ef3b5cabb5 100644
--- a/Doc/library/bisect.rst
+++ b/Doc/library/bisect.rst
@@ -24,6 +24,8 @@ method to determine whether a value has been found.  Instead, the
 functions only call the :meth:`__lt__` method and will return an insertion
 point between values in an array.

+.. _bisect-functions
+
 The following functions are provided:


@@ -140,7 +142,7 @@ thoughts in mind:
 Searching Sorted Lists
 ----------------------

-The above :func:`bisect` functions are useful for finding insertion points but
+The above :ref:`bisect functions <bisect-functions>` are useful for finding insertion points but
```

Reply:
I considered this but abandoned it as it only skips one paragraph -- but will revisit!